.github/workflows/nightly-docs.yml

```diff
@@ -3,7 +3,11 @@ name: Nightly documentation
 on:
   push:
     branches:
-      - '**'
+      - 'dev/feature'
+      - 'dev/patch'
+      - 'enhancement/**'
+      - 'feature/**'
+      - 'fix/**'
```

Comment:
master too, which reflects the current release.

Reply:
> master too, which reflects the current release.

This action is for nightly only which as of now it only reflect latest release therefore it's handled by [release-docs](https://github.com/SkriptLang/Skript/blob/master/.github/workflows/release-docs.yml)